src/core/ext/resolver/dns/native/dns_resolver.c

```diff
@@ -71,9 +72,9 @@ typedef struct {
   /** pending next completion, or NULL */
   grpc_closure *next_completion;
   /** target result address for next completion */
-  grpc_resolver_result **target_result;
+  grpc_channel_args **target_result;
```

Comment:
As per my previous comment, I think "result" is still a fine term here.  The fact that the data type of the result has changed doesn't mean that this isn't still the resolver's result.